cg/meta/workflow/nf_analysis.py

```diff
@@ -904,5 +904,5 @@ def parse_analysis(self, qc_metrics_raw: list[MetricsBase], **kwargs) -> Nextflo
 
     def get_latest_metadata(self, case_id: str) -> NextflowAnalysis:
         """Return analysis output of a Nextflow case."""
-        qc_metrics: list[MetricsBase] = self.get_multiqc_json_metrics(case_id)
+        qc_metrics: list[MetricsBase] = NfAnalysisAPI.get_multiqc_json_metrics(self, case_id)
```

Comment:
I would advise against this, as it will remove the possibility of other nextflow pipelines of override this method

Reply:
That is true, I will have another look at the raredisease method. 👍

Reply:
How about this? I have now reverted the `NfAnalysisAPI.get_latest_metadata` to its original state, so that the other pipelines are unaffected. The `RarediseaseAnalysisAPI.get_multiqc_json_metrics` remains as is and is used for doing the QC check, including the sample pairs. And then in the `RarediseaseAnalysisAPI` I explicitly refer to the `NfAnalysisAPI.get_multiqc_json_metrics` in the `get_latest_metadata` step when collecting the metrics to generate the delivery report, so that it only looks for samples, and not sample pairs.

Reply:
resolve if the new approach works